### DISCUSSION TITLE
art source

### DISCUSSION THREAD

**Discussion by yahavshalem** [romannurik/muzei#759] [Q&A] [answered]
hi, I'm making a similar app for Linux and wanted to know the source of the artwork. does anyone know?

**Comment by ianhanniballake** [comment, marked as answer]
Muzei sources almost all of its 'Featured Art' from [WikiArt](https://www.wikiart.org/), which you can see when you click on many of the images on the [Muzei Archive](https://muzei.co/archive). They do not provide any API you can directly use in an app though, so Muzei hand chose images and built our own backend to serve those images specifically to Muzei - you'd need to do that part yourself.

**Reply by yahavshalem** [reply]
alright, thanks for your help and the quick reply
